cms/templates/asset_index.html

```diff
@@ -42,15 +43,28 @@ <h1 class="page-header">
 
         <nav class="nav-actions" aria-label="${_('Page Actions')}">
             <h3 class="sr">${_("Page Actions")}</h3>
-            <ul>
-                <li class="nav-item">
-                    <a href="#" class="button upload-button new-button"><span class="icon fa fa-plus" aria-hidden="true"></span> ${_("Upload New File")}</a>
-                </li>
-            </ul>
+
+            <form class="filter-criteria-form" action="${asset_callback_url}" method="post">
+                <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+                <ul>
```

Comment:
I don't think this belongs in a ul (the already existing usage of ul/li is weird).